engine/src/python/init.cpp

```diff
@@ -69,6 +69,9 @@ class Unit;
 #define PATHSEP "/"
 #endif
 
+#define Q(x) #x
+#define QUOTE(x) Q(x)
+
 void Python::initpaths() {
```

Comment:
As it turns out, Vega Strike already had a section of code for passing paths to Python. This was it. I added a few lines to this function, and suddenly the code was able to find the base_computer Python scripts.

Reply:
I originally set out to pass actual structs between C++ and python. I actually managed that but then realised that it's probably easier to just pass maps in most cases.

I did look at the original code but was unable to find an example that didn't start with something like 
_RunPython("some_python_code_snippet...");_
whereas I wanted
_RunPython("python_file", "python_function", args);_

Reply:
Maybe I should restate that. Vega Strike already had a function for initializing the Python paths. That's what this was.

I don't have a problem with you adding the `RunPython("python_file", "python_function", args);` function, @royfalk . I think that part makes sense.

Reply:
That's absolutely fine.